app/views/metadata_presenter/component/_autocomplete.html.erb

```diff
@@ -0,0 +1,14 @@
+<%=
+  f.govuk_collection_select component.id.to_sym,
+  component.items,
+  :id,
+  :name,
+  label: { text: input_title },
+  hint:  {
+      data: { "fb-default-text" => default_text('hint') },
+      text: component.hint
+    },
+  style: "width:50%"
+%>
+
+<button type="button" class="fb-govuk-button fb-govuk-button-inverted">Upload options</button>
```

Comment:
```suggestion
<button type="button" class="fb-govuk-button fb-govuk-button-inverted">t('actions.upload_options')</button>
```

Reply:
the locales file is available for use now, so we can add to that whatever is missing as build